tests.cpp

```diff
@@ -279,45 +279,46 @@ TEST_CASE_METHOD(CoherentUnitTestFixture, "test_grover")
     //
     // The target probability here is looking for 100 in the low 8 bits, and
```

Comment:
Comment needs to be adjusted for correctness and clarity.

Reply:
Done.